app/style.js

```diff
@@ -18,11 +18,11 @@ function injectCss(offsetTop, offsetLeft) {
     .gct-file-tree {
       position: absolute;
       top: ${$('#files').offset().top - offsetTop}px;
-      background-color: #fff;
+      background-color: transparent;
       left: ${offsetLeft}px;
       z-index: 28;
       width: 280px;
-      border: 1px solid #ddd;
+      border: 0;
```

Comment:
Without the border looks a bit "random" stuff floating, we could add `border-radius: 3px;` to look like more the GitHub's boxes

Reply:
I mean I removed the border because from a UX perspective giving it the "box-y" appearance makes interfaces feel more closed off, but I can understand the benefit of it since its a scrollable area and you want to communicate to the user what the container is. The bigger change was to remove the hard-coding of the background-color for the container.